components/schemes/Typography/src/MDCTypographyScheme.m

```diff
@@ -15,18 +15,20 @@
 #import "MDCTypographyScheme.h"
 
 #import <CoreText/CoreText.h>
+#import <MaterialComponents/MaterialTypography.h>
 
 @implementation MDCTypographyScheme
 
 - (instancetype)init {
-  return [self initWithDefaults:MDCTypographySchemeDefaultsMaterial201804];
+  return [self initWithDefaults:MDCTypographySchemeDefaultsMaterial201902];
 }
 
 - (instancetype)initWithDefaults:(MDCTypographySchemeDefaults)defaults {
   self = [super init];
   if (self) {
     switch (defaults) {
-      case MDCTypographySchemeDefaultsMaterial201804:
+      case MDCTypographySchemeDefaultsMaterial201804:  // FALLTHRU
```

Comment:
Please create two independent sections within the switch. Having fall-through behavior is more likely than not to introduce bugs in the future.  Each case should fully encapsulate its initialization behavior.

If you prefer, consider creating helper methods or a private subclass as an implementation detail and returning that instead.

Reply:
We should generally try to avoid invoking other methods in an init method.
I intentionally avoid helper methods that only get called from a single place.

Reply:
I have broken the two cases apart and eliminated the fallthru.